### DISCUSSION TITLE
版面分析正确率上不去

### DISCUSSION THREAD

**Discussion by zhangtingyun** [PaddlePaddle/PaddleOCR#13227] [Q&A]
我更换飞浆的版面分析的模型  训练中文数据集 CDLA   但ap一直上不去，一直在80%左右，但官网的准确率不应该在 94%左右吗？
配置如下：
pretrain_weights: https://paddleocr.bj.bcebos.com/ppstructure/models/layout/picodet_lcnet_x1_0_fgd_layout_cdla.pdparams
#pretrain_weights: output/best_model.pdparams
weights: output/picodet_lcnet_x1_0_fgd_layout_cdla/model_final
find_unused_parameters: True
use_ema: true
cycle_epoch: 10
snapshot_epoch: 10
epoch: 10

PicoDet:
  backbone: LCNet
  neck: CSPPAN
  head: PicoHead
  nms_cpu: True

LCNet:
  scale: 1.0
  feature_maps: [3, 4, 5]

metric: COCO
num_classes: 11

TrainDataset:
    name: COCODataSet
    image_dir: train
    anno_path: train.json
    dataset_dir: 'E:\imageTest\CDLA\'
    data_fields: ['image', 'gt_bbox', 'gt_class', 'is_crowd']

EvalDataset:
    name: COCODataSet
    image_dir: val
    anno_path: val.json
    dataset_dir: 'E:\imageTest\CDLA\'

TestDataset:
  !ImageFolder
    anno_path: 'E:\imageTest\CDLA\val.json'

**Comment by SWHL** [comment]
这个不好说。不太清楚官网模型怎么训练的，用了哪些数据集。可能用了也没写。
